packages/components/src/local/organisms/planning-assets/index.tsx

```diff
@@ -71,7 +71,9 @@ export const PlanningAssets: React.FC<PropTypes> = ({
     components={{
       Body: (props): React.ReactElement => {
         if (props.columns.length && onVisibleRowsChange) {
-          onVisibleRowsChange(props.renderData)
```

Comment:
hello @lebaphi  - can you teach me what this fix does please?

Reply:
I think this hook calls immediately when the PlanningChannel is rendering -> reports an error, add setTimeout to make sure the PlanningChannel loads fully, then we call this hook